pkg/migration/migrations/separated_intents.go

```diff
@@ -360,6 +403,18 @@ func (m *migrateLockTablePool) runStatusLogger(ctx context.Context) {
 	}
 }
 
+func countTotalRanges(ctx context.Context, ri rangeIterator) (uint64, error) {
+	var total uint64
+	rs := roachpb.RSpan{Key: roachpb.RKeyMin, EndKey: roachpb.RKeyMax}
+	for ri.Seek(ctx, roachpb.RKeyMin, kvcoord.Ascending); ri.Valid(); ri.Next(ctx) {
```

Comment:
You could replace this with `distSender.CountRanges()`.

Reply:
Heh, I _could_ but I don't have a sender capable of counting in the tests, where we mock it.